views/styles.html

```diff
@@ -192,22 +192,22 @@ <h1 class="base--h2">Form elements</h1>
             <input id="input__text" type="text" placeholder="Text Input" disabled>
             <p class="form--description">This is a description of the input above</p>
           </div>
-          <div class="form--field">
-            <label for="input__password">Password</label>
+          <div class="form--field required--save">
+            <label for="input__password">Password <mark class="mark-save">required to publish</mark></label>
             <input id="input__password" type="password" placeholder="Type your Password" >
             <p class="form--description">This is a description of the input above</p>
           </div>
-          <div class="form--field">
-            <label for="input__webaddress">Web Address</label>
+          <div class="form--field required--publish">
+            <label for="input__webaddress">Web Address <mark class="mark-publish">required to publish</mark></label>
             <input id="input__webaddress" type="url" placeholder="http://yoursite.com">
             <p class="form--description">This is a description of the input above</p>
           </div>
-          <div class="form--field">
-            <label for="input__emailaddress">Email Address</label>
+          <div class="form--field required--publish">
```

Comment:
Why does the form field get `required--*` class? There's no reason for that. Please remove them